libraries/AP_Logger/AP_Logger_MAVLinkLogTransfer.cpp

```diff
@@ -174,11 +174,16 @@ void AP_Logger::handle_log_request_erase(GCS_MAVLINK &link, const mavlink_messag
 void AP_Logger::handle_log_request_end(GCS_MAVLINK &link, const mavlink_message_t &msg)
 {
     WITH_SEMAPHORE(_log_send_sem);
-    mavlink_log_request_end_t packet;
-    mavlink_msg_log_request_end_decode(&msg, &packet);
+    // mavlink_log_request_end_t packet;
+    // mavlink_msg_log_request_end_decode(&msg, &packet);
+    end_log_transfer();
+}
 
+void AP_Logger::end_log_transfer()
+{
     transfer_activity = TransferActivity::IDLE;
     _log_sending_link = nullptr;
+    backends[0]->end_log_transfer();
```

Comment:
Shouldn't you call this on all of the backends?

Reply:
> Shouldn't you call this on all of the backends?

We only do transfers from the first backend - there's lots of this [0] stuff around.